src/VisualStudio/Core/Def/TableDataSource/AbstractTableEntriesSnapshot.cs

```diff
@@ -1,303 +0,0 @@
-﻿// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-// See the LICENSE file in the project root for more information.
-
-using System;
-using System.Collections.Immutable;
-using System.Diagnostics.CodeAnalysis;
-using System.IO;
-using System.Linq;
-using System.Threading;
-using System.Windows;
-using System.Windows.Controls;
-using Microsoft.CodeAnalysis;
-using Microsoft.CodeAnalysis.Diagnostics;
-using Microsoft.CodeAnalysis.Editor.Shared.Utilities;
-using Microsoft.CodeAnalysis.Navigation;
-using Microsoft.CodeAnalysis.Text;
-using Microsoft.VisualStudio.Shell.TableManager;
-using Microsoft.VisualStudio.Text;
-using Roslyn.Utilities;
-
-namespace Microsoft.VisualStudio.LanguageServices.Implementation.TableDataSource;
-
-/// <summary>
-/// Base implementation of ITableEntriesSnapshot
-/// </summary>
-internal abstract class AbstractTableEntriesSnapshot<TItem> : ITableEntriesSnapshot
```

Comment:
this abstract class had no subclasses.  So you can never have an instance of this class.  Deleting outright.  The remainder is fallout of htat.